### DISCUSSION TITLE
Cannot receive data from Tomato collector

### DISCUSSION THREAD

**Discussion by dotcode** [NightscoutFoundation/xDrip#2345] [Q&A]
Hi there, I'm using:
- Libre 2
- MiaoMiao 1
- Pixel 6 (recently upgraded to Android 13)
- xdrip+

Recently I've been unable to receive data from the Tomato collector, as illustrated in the image below. Do you know what could be the cause?

![Screenshot_20220919-211342](https://user-images.githubusercontent.com/1267506/191109758-629b1aa8-be35-42ad-b399-1c2dade9b358.png)
![Screenshot_20220919-211605](https://user-images.githubusercontent.com/1267506/191109767-d1d90f18-297b-4d09-b6fa-90bca019c0c3.png)

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Please post the BT device tab of system status
